converttodzi.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
It uses sharp to convert any compatible image files it finds in the `collection-items` directory to dzi. It generates their respective tiles too, you can find them in folders named after the original files and postfixed with `_files`. If you open the network tab in Chrome dev tools, you'll see that it successfully calls the appropriate tiles with status 200 everytime you zoom-in. I had to use a high-res image to properly show proof-of-concept; the original images (the aircraft carrier for example) weren't high-res enough that Nick and I initially thought it wasn't calling the right tiles etc.